### PR TITLE
Google Threat Intelligence Solution

### PR DESCRIPTION
**Pull request by NicoRomero-07**
Change(s):
   - New Custom Connector
   - New Playbooks for Enrichment

   Reason for Change(s):
   - New Google Threat Intelligence Solution

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Need Help
   
  IDs Should Be Derived From ResourceIDs     x2                                                                           
    [-] IDs Should Be Derived From ResourceIDs (38 ms)                                                                  
        Property: "id" must use one of the following expressions for an resourceId property:                            
extensionResourceId,resourceId,subscriptionResourceId,tenantResourceId,if,parameters,reference,variables,subscription,guid

These are the ids: 
"id": "[variables('_playbookcontentProductId1')]"
 "id": "[variables('_solutioncontentProductId')]"

And variables:
"_playbookcontentProductId1": "[concat(take(variables('_solutionId'),50),'-','lc','-', uniqueString(concat(variables('_solutionId'),'-','LogicAppsCustomConnector','-',variables('_playbookContentId1'),'-', variables('playbookVersion1'))))]",
"_solutioncontentProductId": "[concat(take(variables('_solutionId'),50),'-','sl','-', uniqueString(concat(variables('_solutionId'),'-','Solution','-',variables('_solutionId'),'-', variables('_solutionVersion'))))]"
           
Id definition is using variables, shouldn't that be enough to meet the requirement?

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @NicoRomero-07, Please update the branch from master once. The checks are stucked

File: Solutions/Google Threat Intelligence/Data/Solution_GoogleThreatIntelligence.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/GoogleThreatIntelligence.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "This Google Threat Intelligence Solution contains Playbooks that can help enrich incident information with threat information and intelligence for IPs, file hashes and URLs from Google Threat Intelligence. Enriched information can help drive focused investigations in Security Operations.",
+  "Data Connectors": [],
+  "Parsers": [],
+  "Hunting Queries": [],
+  "Analytic Rules": [],
+  "Workbooks": [],
+  "Playbooks": [
+    "Playbooks/CustomConnector/GTICustomConnector/azuredeploy.json",
+	  "Playbooks/Playbooks/GTIEnrichment/GTI-EnrichIncident/azuredeploy.json",
```
Comment (v-prasadboke) [inline Solutions/Google Threat Intelligence/Data/Solution_GoogleThreatIntelligence.json:13]: Please correct the file path of playbooks in input file

File: Solutions/Google Threat Intelligence/Playbooks/GTIEnrichment/GTI-EnrichAlert/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+        "title": "Google Threat Intelligence - IOC Enrichment",
+        "description": "This playbook will enrich IP, Hash, URL & Domain entities found in alerts.",
+        "prerequisites": [
+            "You will need to register to Google Threat Intelligence for an API key"
+        ],
+        "postDeployment": [
+            ""
+        ],
+        "prerequisitesDeployTemplateFile": "",
+        "lastUpdateTime": "",
```
Comment (v-prasadboke) [inline Solutions/Google Threat Intelligence/Playbooks/GTIEnrichment/GTI-EnrichAlert/azuredeploy.json:14]: Please add last update time to all the playbooks

**Comment by v-prasadboke** [comment]
Hello @NicoRomero-07, I would suggest to add deployment steps of custom connector in prerequisites of playbooks

**Comment by v-prasadboke** [comment]
other than this all looks good to me

**Comment by NicoRomero-07** [comment]
> Hello @NicoRomero-07, I would suggest to add deployment steps of custom connector in prerequisites of playbooks
Hello @v-prasadboke ,
In the content hub, the CustomConnector won't appear for users, right? That is, they will have to deploy the custom connector by the URI of the GitHub repository to the azuredeploy.json file. We can make it easier to use by providing a 'Deploy to Azure' button.

**Comment by v-prasadboke** [comment]
Hello @NicoRomero-07, you missed on packaging the solution. 
Can you repackage it

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**